test/scripts/console/new.js

```diff
@@ -1,215 +1,233 @@
 'use strict';
 
-const fs = require('hexo-fs');
+const { exists, mkdirs, readFile, rmdir, unlink } = require('hexo-fs');
 const moment = require('moment');
-const pathFn = require('path');
+const { join } = require('path');
 const Promise = require('bluebird');
-const sinon = require('sinon');
+const { useFakeTimers } = require('sinon');
 
 describe('new', () => {
   const Hexo = require('../../../lib/hexo');
-  const hexo = new Hexo(pathFn.join(__dirname, 'new_test'), {silent: true});
+  const hexo = new Hexo(join(__dirname, 'new_test'), {silent: true});
   const n = require('../../../lib/plugins/console/new').bind(hexo);
   const post = hexo.post;
   const now = Date.now();
   let clock;
 
-  before(() => {
-    clock = sinon.useFakeTimers(now);
+  before(async () => {
+    clock = useFakeTimers(now);
 
-    return fs.mkdirs(hexo.base_dir).then(() => hexo.init()).then(() => hexo.scaffold.set('post', [
+    await mkdirs(hexo.base_dir);
+    await hexo.init();
+    await hexo.scaffold.set('post', [
       'title: {{ title }}',
       'date: {{ date }}',
       'tags:',
       '---'
-    ].join('\n'))).then(() => hexo.scaffold.set('draft', [
+    ].join('\n'));
+    await hexo.scaffold.set('draft', [
       'title: {{ title }}',
       'tags:',
       '---'
-    ].join('\n')));
+    ].join('\n'));
```

Comment:
```js
await Promise.all([
  hexo.scaffold.set('post', [
    'title: {{ title }}',
    'date: {{ date }}',
    'tags:',
    '---'
  ].join('\n')),
  hexo.scaffold.set('draft', [
    'title: {{ title }}',
    'tags:',
    '---'
  ].join('\n')),
])
```

Could it be done like this? Should have better performance, although this is in a unit test.